app/frontend/layouts/app_router/components/AppRouter__Nav.res

```diff
@@ -17,7 +17,7 @@ let renderLinks = (courses, selectedPage) => {
   | Page.SelectedCourse(courseId, coursePage) =>
     let currentCourse = ArrayUtils.unsafeFind(
       course => Course.id(course) == courseId,
-      "Could not find currentCourse with ID " ++ courseId,
+      t("current_course_not_found") ++ courseId,
```

Comment:
`current_course_not_found` is used in 3 files; It will be good to move it into the shared group; Replace all instances in this file.